### DISCUSSION TITLE
How to setup memory config for Next.js SSR pages?

### DISCUSSION THREAD

**Discussion by hcentelles** [vercel/vercel#4537] [Help]
The docs said that `functions` config param only works for functions inside `/api` folder, that for other cases you should use `builds`.  I’ve tried that without luck. Anyone knows how to rise the memory allocation for SSR pages?

**Comment by mcsdevv** [comment]
Hello @hcentelles! I have replied to your ticket, as stated, it would be great to understand your use case for this, would you be able to share that? We recommend going static-first in nearly every scenario so it'd be great to learn more.
